spacy/tokens/span_group.pyx

```diff
@@ -4,7 +4,13 @@ import srsly
 
 from spacy.errors import Errors
 from .span cimport Span
-from libc.stdint cimport uint64_t, uint32_t, int32_t
+from ..structs cimport SpanC
+
+from libcpp.algorithm cimport sort as sort_vector
+from libcpp.set cimport set
+from libcpp.utility cimport pair
+
+from typing import Iterable, Tuple, Union, Optional, TYPE_CHECKING
```

Comment:
You couldn't have known this, but we typically put the imports from `typing` at the very first line

Reply:
Done :)